File: MODUL 4 C/PRAK304-2210817310004-Muhammad Adika Rizqullah.c
#include <stdio.h>

int main()
{
    for(int i = 1; i<=5; i++){
    int a;
    scanf("%d", &a);

    if(a==0){
        printf("Nol\n", a);
    }
    else if(a<=9){
        printf("Satuan\n", a);
    }
    else if (a<=19) {
            printf("Belasan\n", a);
    }
    else if (a>=100){
        printf("Anda Menginput Melebihi Limit Bilangan\n", a);
    }
    else {
        printf("Puluhan\n");
    }
    }
}

File: MODUL 4 PYTHON/PRAK304-2210817310004-Muhammad Adika Rizqullah.py
for x in range(5):
    nilai = int(input(""))

    if nilai == 0: 
        print("Nol\n")
    elif nilai <= 9: 
        print("Satuan\n")
    elif 11>= nilai <=19: 
        print("Belasan\n")
    elif nilai >= 100: 
        print("Anda Menginput Melebihi Limit\n")
    else: 
        print("Puluhan\n")